### PR TITLE
Fix the build for the fatJar plugin + jcommander upgrade

### PR DESCRIPTION
**Pull request by nbauma109**
I noticed that the fatJar was incompatible with gradle 4+ because they stopped supporting a feature it was using.

I downgraded to gradle 3.5.1. It depends if you prefer the benefits of a newer gradle or the fatJar plugin. Let me know.

Also noticed that with the old jcommander version, the gradle build was looking for a jcommander project inside my own repo as I tried to use procyon as a dependency with jitpack, so I upgraded to a newer version.

### CODE REVIEW

**Comment by mstrobel** [comment]
Thanks for the PR!  Are you aware of any suitable replacement for the fatJar plugin that would allow me to upgrade to a current gradle version?

**Comment by nbauma109** [comment]
ShadowJar could be a good replacement https://github.com/johnrengelman/shadow

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #Sun Oct 27 14:39:59 EDT 2019
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5.1-all.zip
```
Comment (liach) [inline gradle/wrapper/gradle-wrapper.properties:2]: Hmm, rolling back gradle isn't particularly a good idea. You should really seek alternatives like the shadow plugin from johnrengelman

Comment (mstrobel) [reply gradle/wrapper/gradle-wrapper.properties:2]: Indeed, I'm tweaking this now and will push shortly.

**Pull request merged**